src/nni_manager/core/nnimanager.ts

```diff
@@ -180,13 +180,19 @@ class NNIManager implements Manager {
         return this.experimentProfile.id;
     }
 
+    public async viewExperiment(): Promise<void> {
+        this.log.info(`Viewing experiment: ${this.experimentProfile.id}`);
+        this.experimentProfile = await this.dataStore.getExperimentProfile(getExperimentId());
+        this.setStatus('VIEWING');
```

Comment:
in VIEWING mode, do we need to disable some buttons in webui and disable some functionalities in `nnictl`?